client/history/client.go

```diff
@@ -29,14 +29,14 @@ import (
 	"sync"
 	"time"
 
-	"go.temporal.io/api/serviceerror"
 	"google.golang.org/grpc"
 
 	"go.temporal.io/server/api/historyservice/v1"
 	replicationspb "go.temporal.io/server/api/replication/v1"
 	"go.temporal.io/server/common"
 	"go.temporal.io/server/common/log"
 	"go.temporal.io/server/common/log/tag"
+	serviceerrors "go.temporal.io/server/common/serviceerror"
```

Comment:
The last `s` in `serviceerrors` is annoying but it doesn't mean plural form. It stands for `server` the same as `replication_s_pb` few lines above.